cicd/crabtaskworker_pypi/Dockerfile

```diff
@@ -68,22 +67,30 @@ RUN mkdir /build
 WORKDIR /build
 
 # install dependencies
-COPY --chown=${USER}:${USER} --from=wmcore-src /wmcore_version ./
-COPY --chown=${USER}:${USER} cicd/crabtaskworker_pypi/requirements.txt ./
+COPY cicd/crabtaskworker_pypi/requirements.txt ./
 RUN pip install -r requirements.txt \
-    && pip install --no-deps wmcore==$(cat wmcore_version) \
     && pip cache purge
 
-# copy htcondor config
-RUN mkdir /etc/condor
-COPY --chown=${USER}:${USER} cicd/crabtaskworker_pypi/condor_config /etc/condor/
+# create install dir
+RUN install -d -o ${USER} -g ${USER} ${WDIR}/srv/current/lib/python/site-packages
+
+# install wmcore
+COPY cicd/crabserver_pypi/wmcore_requirements.txt \
+     cicd/crabserver_pypi/installWMCore.sh \
+     ./
+RUN ./installWMCore.sh wmcore_requirements.txt ${WDIR}/srv/current/lib/python/site-packages
```

Comment:
is this "install it again after it was installed at Line 16 ?

Reply:
Line 16 is git clone for packaging the files in `data` directory.
Yes, we do clone twice, but it is the same rep and same tag (or branch) so it should be fine.
Does it twice to avoid complicate logic in Dockerfile.

Reply:
thanks